### PR TITLE
transform gamepad pose by sittingToStandingTransform

### PR DESCRIPTION
**Pull request by beaucronin**
The library currently does not work with Vive controllers because the gamepad pose must be transformed by the `stageParameters.sittingToStandingTransform` in order to be placed correctly (see #13). This is straightforward, but it does mean that the VRDisplay object must be available in the update function. This PR achieves this by simply adding it as a constructor argument, but some other method might be more elegant.

I've also updated the example to accept the relevant arguments, and verified that these changes work on my own system under latest Chromium. I have not tested with Touch, since I don't yet have hardware available.

### CODE REVIEW

**Comment by borismus** [comment]
Can we get the actively presenting VR Display within the RayInput library to avoid passing in the VRDisplay? It would be better to keep the simple signature if possible.

File: src/ray-input.js [hunk 1]
```diff
@@ -23,12 +23,13 @@ import InteractionModes from './ray-interaction-modes'
  * API wrapper for the input library.
  */
 export default class RayInput extends EventEmitter {
-  constructor(camera, opt_el) {
+  constructor(camera, opt_el, vrDisplay) {
```
Comment (borismus) [inline src/ray-input.js:26]: If we do need the extra param, could we make the opt_el parameter be the last one?

**Comment by kschzt** [comment]
@borismus @beaucronin there is already the retrieval of VRDisplays in https://github.com/borismus/ray-input/blob/master/src/ray-controller.js#L68, could we move that to RayInput and pass along? either that, or remove the one in RayController and use this one

**Comment by borismus** [comment]
Bump

**Comment by kschzt** [comment]
I think the latter option would better avoid new promises and events

**Comment by beaucronin** [comment]
Sorry, just cleared out a deluge of notifications and saw this. Will take a look at @kschzt's suggestion.

**Comment by borismus** [comment]
Bump

**Pull request closed without merging**